lib/app.js

```diff
@@ -74,7 +74,20 @@ export class Container extends Component {
 
   render () {
     const {children} = this.props
-    return <>{children}</>
+
+    // make sure to support all React-like libraries, whether they support Arrays, React.Fragment or not
+    if (Array.isArray(children)) {
+      return children.map((child, index) => {
+        // only set key if it's null or undefined
+        if (child.key == null) {
```

Comment:
@timneutkens 
We could also just use a false check, though values like `0` would match that too incorrectly.